tasks/init.json

```diff
@@ -0,0 +1,52 @@
+{
+  "description": "Make a HTTP or HTTPS request.",
+  "parameters": {
+    "body": {
+      "description": "The request body.",
+      "type": "Optional[Data]"
+    },
+    "body_format": {
+      "description": "The format of the request body. Automatically attempts to convert the body to the specified format.",
+      "type": "Enum[form-urlencoded, json, raw]",
+      "default": "json"
+    },
+    "cacert": {
+      "description": "An absolute path to the CA certificate.",
+      "type": "Optional[String[1]]"
+    },
+    "cert": {
+      "description": "An absolute path to the client certificate.",
+      "type": "Optional[String[1]]"
+    },
+    "headers": {
+      "description": "A map of headers to add to the payload.",
+      "type": "Optional[Hash]"
+    },
+    "key": {
+      "description": "An absolute path to the RSA keypair.",
+      "type": "Optional[String[1]]"
+    },
+    "method": {
+      "description": "The HTTP method to use.",
+      "type": "Enum[delete, get, post, put]",
+      "default": "get"
+    },
+    "success_codes": {
+      "description": "A list of status codes that indicate the task should return a successful result.",
+      "type": "Array[Numeric]",
+      "default": [200]
+    },
+    "url": {
```

Comment:
Went with `url` over a combination of `base_url` and `path`, as it seems simpler to only have a single parameter that handles the URL and plans can use interpolation to accomplish what `base_url` and `path` would.

Reply:
The purpose of that spec was to support remote targets where the url is configured in inventory and the path is specified for each invocation.

Reply:
But wouldn't that be under `_target` then?

Reply:
Ah meaning you would only pass `path` as a task parameter and omit `baseurl`, opting instead to use what was specified in `_target`?